misc/ls-extensions/modules/partial-parser/src/main/java/io.ballerina.parsers/PartialParserClientCapabilities.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.ballerina.parsers;
+
+import org.ballerinalang.langserver.commons.registration.BallerinaClientCapability;
+
+/**
+ * Client capabilities for the jsonToRecord service.
+ *
+ * @since 2.0.0
```

Comment:
@IMS94 Should the version be 2201.1.1?

Reply:
Yes

Reply:
Fixed with 9bf9ee11894fa6e30e5c97e65a867c2f941b3710